source/about/mattermost-v9-changelog.md

```diff
@@ -1,11 +1,118 @@
+ - Added minor performance improvements to webapp initialization.
+
+#### mmctl
+ - Added two new commands to mmctl, ``mmctl job list`` and ``mmctl job update``.
+ - Panic message is now printed when mmctl panics.
+ - Setting ``AdvancedLoggingJSON`` via mmctl is now supported.
+
+### Bug Fixes
+ - Fixed an issue that displayed a wrong count for custom group members on the notification warning.
+ - Fixed a panic when the password was too long.
+ - Fixed an issue where configuration patches through mmctl did not correctly merge plugin configuration values.
+ - Fixed issues with the OpenID local development.
+ - Fixed an issue where Latex was not rendered in a code block as code when Latex rendering was disabled.
+ - Fixed an issue with saving custom roles.
+ - Fixed an issue with the left-hand side scrollbar auto-hide functionality for Chrome and Safari.
+ - Fixed Group Message to private channel conversion edge cases.
+ - Fixed an issue where users with the user management permission were unable to view the list of users in the **System Console > Users** page.
+ - Fixed more web app performance reports being marked as outdated after a user's computer woke up from sleep.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added ``TerminateSessionsOnPasswordChange`` to configure the sessions revocation during password resets.
+
+#### Changes to the Enterprise plan:
+ - Under ``ElasticsearchSettings`` in ``config.json``:
+    - Added ``Backend`` to differentiate between Elasticsearch and Opensearch. The default value is Elasticsearch.
+
+### API Changes
+ - Added new API endpoints to manage remote clusters.
+ - Added two new query parameters to ``GET /api/v4/jobs, job_type`` and status.
+ - Added a new endpoint ``PATCH /api/v4/jobs/{job_id}/status``.
+ - Updated ``AddChannelMember`` to accept a list of userIds.
+ - Added six new permissions to manage the status of particular jobs:
+     - ``PermissionManagePostBleveIndexesJob``
+     - ``PermissionManageDataRetentionJob``
+     - ``PermissionManageComplianceExportJob``
+     - ``PermissionManageElasticsearchPostIndexingJob``
+     - ``PermissionManageElasticsearchPostAggregationJob``
+     - ``PermissionManageLdapSyncJob``
+
+### Go Version
+ - v9.11 is built with Go ``v1.21.8``.
+
+### Open Source Components
+ - Removed ``stylelint``, and added ``elastic/go-elasticsearch`` to https://github.com/mattermost/mattermost/.
+
+### Known Issues
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
```

Comment:
@amyblais just curious as we will be looking more into push notifications issues, do we have a ticket for this one?